docs/userguide/networking/work-with-networks.md

```diff
@@ -499,6 +499,130 @@ environment variables into a running container without significant effort and he
 it is not compatible with `docker network` which provides a dynamic way to connect/
 disconnect containers to/from a network.
 
+### Network-scoped alias
+
+While `links` provide private name resolution that is localized within a container,
+the network-scoped alias provides a way for a container to be discovered by an
+alternate name by any other container within the scope of a particular network.
+Unlike the `link` alias, which is defined by the consumer of a service, the
```

Comment:
Yes. we are trying to differentiate the `alias` functionality provided by `link` and `--net-alias`. Infact we call it out in the docs as `alias` for link as well. So, it is better to be explicit here.